frontend/index.html

```diff
@@ -42,12 +48,16 @@
     <link rel="stylesheet" href="./js/components/ListTable/ListTable.component.css">
     <link rel="stylesheet" href="./js/components/LoadingSpinner/LoadingSpinner.component.css">
 </body>
-    
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
-    <script src="https://unpkg.com/popper.js@1.12.6/dist/umd/popper.js" integrity="sha384-fA23ZRQ3G/J53mElWqVJEGJzU0sTs+SvzG8fXVWP+kJQ1lwFAOkcUOysnlKJC33U" crossorigin="anonymous"></script>
-    <script src="https://unpkg.com/bootstrap-material-design@4.1.1/dist/js/bootstrap-material-design.js" integrity="sha384-CauSuKpEqAFajSpkdjv3z9t8E7RlpJ1UP0lKM/+NdtSarroVKu069AlsRPKkFBz9" crossorigin="anonymous"></script>
-    <script type="text/javascript" src="third_party/jquery.dataTables.min.js"></script>
-    <script src='https://kit.fontawesome.com/a076d05399.js'></script>
-    <script type="module" src="js/app.js"></script>
+
+<!-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script> -->
```

Comment:
please remove if not needed

Reply:
will remove this line.

Reply:
Please remove the commented code before merging

Reply:
removed.